Code review

mpociot opened the pull request:
I added and modified a few things:

* Changed the Message method names to omit the `set`, to make it equal to the other message classes
* Used the recipient number from the `routeNotificationForCmsms` method
* Removed PHP 7 `string` typehint
* Added available message methods to README
* Validate the "originator"
* Minor code style fixes


If you have any questions or comments, let me know.

mbardelmeijer commented on src/CmsmsClient.php:
```diff
@@ -33,18 +29,17 @@ public function __construct(GuzzleClient $client, $productToken)
     {
-        if (empty($message->originator)) {
+        if (is_null(Arr::get($message->toXmlArray(), 'FROM'))) {
             $message->setOriginator(config('services.cmsms.originator'));
```
Should be `$message->originator(config('services.cmsms.originator'));` due to the removed `set` prefixes in the message class.

mbardelmeijer commented:
Nice work :)

Noticed the `$recipient` variable has moved to the client instead of the message. Care to elaborate on that? Is it to support the `routeNotificationForCmsms` method? If so, the recipient information in the message class should be removed as it's unused at the moment.

mpociot commented:
Yes, I moved the recipient variable to the client, because otherwise the routeNotificationFor method was not used at all.
You're right. In this case the message does not need the recipient property. 

To be honest, I'm not a big fan of the way this is managed in the Facebook package, because I think that Laravels routeNotificationFor method is there for a reason ;)

I will change the method naming you mentioned and remove the recipient property tomorrow.

mpociot commented:
Hi @mbardelmeijer ,

I removed the recipient property from the Message class and changed the `setOriginator` method call to the correct signature.
I also added a test to cover the case when the default originator should be used.

mbardelmeijer commented:
Cool! Could you also remove the `recipient` setter in the message class & remove the corresponding tests? Test are currently failing. After that, LGTM :+1:

mpociot commented:
Whoops, removed it.
Sorry for that - seems like I need more ☕

mbardelmeijer commented:
Awesome, thanks!
